lib/grape/serve_file/file_body.rb

```diff
@@ -0,0 +1,32 @@
+module Grape
+  module ServeFile
+    # Class helps send file through API
+    class FileBody
+      attr_reader :path
+
+      # @param path [String]
+      def initialize(path)
+        @path = path
+      end
+
+      # Need for Rack::Sendfile middleware
+      #
+      # @return [String]
+      def to_path
+        path
+      end
+
+      def each
+        File.open(path, 'rb') do |file|
+          while (chunk = file.read(16_384))
```

Comment:
Should probably be extracted as a constant, just in case someone wants to update it at their runtime.